compiler/src/iree/compiler/Bindings/Native/Transforms/WrapEntryPoints.cpp

```diff
@@ -154,19 +141,66 @@ static LogicalResult wrapImportFunc(mlir::ModuleOp moduleOp,
   return success();
 }
 
+// Populates attributes on |wrapperOp| to support runtime reflection.
+static void populateReflectionAttrs(IREE::ABI::InvocationModel invocationModel,
+                                    func::FuncOp exportOp,
+                                    func::FuncOp wrapperOp) {
+  auto *context = exportOp.getContext();
+  SmallVector<NamedAttribute, 4> attrs;
```

Comment:
Can you add a comment here that we are assembling exported reflection attrs? I had to read this a couple of times and misread it as assembling new function attrs first.

Reply:
Any specific phrasing you want over what's there?
```c++
// Populates attributes on |wrapperOp| to support runtime reflection.
static void populateReflectionAttrs(IREE::ABI::InvocationModel invocationModel,
                                    func::FuncOp exportOp,
                                    func::FuncOp wrapperOp) {
```
I think this is fairly clear that it's constructing reflection attrs for runtime reflection, but maybe you'd like some examples?

Reply:
Changed the wording a bit to mention how to look them up at runtime.